torchgeo/trainers/regression.py

```diff
@@ -93,13 +93,17 @@ def __init__(self, **kwargs: Any) -> None:
             in_channels: Number of input channels to model
             learning_rate: Learning rate for optimizer
             learning_rate_schedule_patience: Patience for learning rate scheduler
-            freeze_backbone: Fine-tune the cls head by freezing the backbone
+            freeze_backbone: Freeze the backbone network to linear probe
+                the regression head
```

Comment:
Same comment for backbone

Reply:
If you can finish this that would be great.

Reply:
Nvm it's updated now